singer/messages.py

```diff
@@ -180,13 +182,20 @@ def parse_message(msg):
     # lossy conversions.  However, this will affect
     # very few data points and we have chosen to
     # leave conversion as is for now.
-    obj = json.loads(msg)
+    obj = json.loads(msg, use_decimal=True)
```

Comment:
@psantacl Why was this change made? This broke one of our targets because it was resulting in "Object of type Decimal is not JSON serializable" exceptions when trying to do `json.dump` on the obj. Pinning the singer-python version to 5.7.0 resolved this, but I ended up spending a lot of time tracking this down. Also, it doesn't look like the comments above this line are accurate anymore.

Reply:
This change was made to preserve precision on floating point numbers since python floats use binary floating point and thus can't  accurately represent all numbers.

I think this is easily fixable by just using `simplejson.dumps` in your target.

You are correct that the comment is outdated, I'll make a note to clean it up.  Thanks!